### ISSUE TITLE
Fresh install problems

### ISSUE THREAD

**Issue by TomaszSkorupka** [Sylius/Sylius#2814]
`composer create-project sylius/sylius -s dev`

Everything ok.

`app/console sylius:install`

```
Step 3 of 4. Shop configuration.
Please enter list of locale codes, separated by commas or just hit ENTER to use "en_US". For example "en_US, de_DE".
In which language your customers can browse the store?  
Adding angielski (Stany Zjednoczone).
PHP Fatal error:  Uncaught exception 'Doctrine\ORM\NonUniqueResultException' in /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:623
Stack trace:
#0 /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/src/Sylius/Bundle/ResourceBundle/Doctrine/ORM/EntityRepository.php(78): Doctrine\ORM\AbstractQuery->getOneOrNullResult()
#1 /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/EntityRepository.php(241): Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository->findOneBy(Array)
#2 /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php(151): Doctrine\ORM\EntityRepository->__call('findOneByCode', Array)
#3 /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php(151): Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository->findOneByCode('en_US')
#4 /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/src/Sylius/Bundle/In in /media/tomasz/DATA/webdev/2015/wemakeb/sylius/sylius/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php on line 623
```

**Comment by vdaguenet** [comment]
I am experiencing the same issue with en_US

**Comment by pjedrzejewski** [comment]
Should be fixed by #2852.
